_posts/2019-05-31-moveit2-alpha-release.md

```diff
@@ -0,0 +1,68 @@
+---
+author: RobColeman
+comments: false
+date: 2019-05-31
+layout: post
+title: Announcing MoveIt 2 Alpha Release
+media_type: image
+media_link: /assets/images/moveit2_alpha.jpg
```

Comment:
Under assets/images let's make a folder called ``blog_posts`` and start keeping all blog-related images there. Otherwise this folder will continue to get messy.